### ISSUE TITLE
set zoom class on page load

### ISSUE THREAD

**Issue by trvrm** [dagjomar/Leaflet.ZoomCSS#2]
You might want to add a call to ```_zoomCSS``` in ```addHooks()``` so the zoom class gets added on page load: at the moment it doesn't exist until the user actually performs a zoom.  Something like:

```
  addHooks: function () {
      this._zoomCSS(); 
      this._map.on('zoomend', this._zoomCSS, this);
  },...```

**Comment by dagjomar** [comment]
@trvrm You want to make a pull request with that?

**Comment by trvrm** [comment]
https://github.com/dagjomar/Leaflet.ZoomCSS/pull/3

**Comment by dagjomar** [comment]
Fixed in #3
